File: NotificationSystem/NotificationSystem.Web/Controls_Search/ctrSearch_TT_Find.ascx.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;

namespace NotificationSystem.NotificationSystem.Web.Controls_Search
{
    public partial class ctrSearch_TT_Find : System.Web.UI.UserControl
    {
        private int m_TroubleTicketNo = 0;

        public int TroubleTicketNo
        {
            get
            {
                return m_TroubleTicketNo;
            }
            set
            {
                m_TroubleTicketNo = value;
            }
        }
        protected void Page_Load(object sender, EventArgs e)
        {

        }

        public void PopulateSearchControl()
        {
            m_TroubleTicketNo = int.Parse(this.txttroubleticketno.Text);
        }
    }
}

File: NotificationSystem/NotificationSystem.Web/Controls_Search/ctrSearch_TT_Find.ascx.vb
Imports System
Imports System.Web.UI

Namespace NotificationSystem.NotificationSystem.Web.Controls_Search
    Public Partial Class ctrSearch_TT_Find
        Inherits UserControl
        Private m_TroubleTicketNo As Integer = 0

        Public Property TroubleTicketNo As Integer
            Get
                Return m_TroubleTicketNo
            End Get
            Set(value As Integer)
                m_TroubleTicketNo = value
            End Set
        End Property
        Protected Sub Page_Load(sender As Object, e As EventArgs)

        End Sub

        Public Sub PopulateSearchControl()
            m_TroubleTicketNo = Integer.Parse(txttroubleticketno.Text)
        End Sub
    End Class
End Namespace
